CHANGES.txt

```diff
@@ -28,8 +28,11 @@
 * Allow fine grained control over the use of wheels and source builds.
   (:pull:`2699`)
 
+* ``--no-use-wheel`` and ``--use-wheel`` are deprecated in favour of new
+  options ``--no-binary`` and ``--only-binary``. (:pull:`2699`)
```

Comment:
thanks, but worried people will see this and simply try to replace `--no-use-wheel` with `--no-binary`, which won't work.  I know this is the changelog, and not docs, but maybe a few lines to prevent that confusion.